robottelo/utils/virtwho.py

```diff
@@ -526,3 +526,25 @@ def create_http_proxy(org, name=None, url=None, http_type='https'):
         organization=[org.id],
     ).create()
     return http_proxy.url, http_proxy.name, http_proxy.id
+
+
+def get_configure_command_option(deploy_type, args, org=DEFAULT_ORG):
+    """Return the deploy command line based on option.
+    :param str option: the unique id of the configure file you have created.
+    :param str org: the satellite organization name.
+    """
+    username, password = Base._get_username_password()
+    if deploy_type == 'location-id':
+        return "hammer -u {} -p {} virt-who-config deploy --id {} --location-id '{}' ".format(
+            username, password, args['id'], args['location-id']
```

Comment:
use `f-string`

Reply:
@jyejare Thanks for your suggestion. resolve on the commit https://github.com/SatelliteQE/robottelo/commit/78f472a03ffa50b3b9f524c35f78acd17b26bc0a